src/main/java/uk/gov/hmcts/reform/userprofileapi/controller/UserProfileController.java

```diff
@@ -68,36 +70,36 @@
 
 
     @ApiOperation(value = "Create a User Profile",
-                  authorizations = {
+            authorizations = {
                     @Authorization(value = "ServiceAuthorization"),
-                    @Authorization(value = "Authorization") 
-                  })
+                    @Authorization(value = "Authorization")
+            })
     @ApiResponses({
-        @ApiResponse(
-            code = 201,
-            message = "Create a User Profile using request body",
-            response = UserProfileCreationResponse.class
-        ),
-        @ApiResponse(
-            code = 400,
-            message = "Bad Request",
-            response = String.class
-        ),
-        @ApiResponse(
-            code = 409,
-            message = "User already exists",
-            response = String.class
-        ),
-        @ApiResponse(
-            code = 500,
-            message = "Internal Server Error",
-            response = String.class
-        )
+            @ApiResponse(
+                    code = 201,
+                    message = "User Profile created successfully",
+                    response = UserProfileCreationResponse.class
+            ),
+            @ApiResponse(
+                    code = 400,
+                    message = "An invalid request has been provided",
+                    response = String.class
```

Comment:
403 status code missing

Reply:
updateUserProfile() api add 412 with message which i sent you in the slack

Reply:
These have been addressed in the following commits:

- 403: https://github.com/hmcts/rd-user-profile-api/pull/302/commits/2b89ed9cd7068384338d596c65af7faf8429ca35

- 412: https://github.com/hmcts/rd-user-profile-api/pull/302/commits/5e39cb1aad935768aec988c11783e603837feb0e